src/lib/rectangleTool.ts

```diff
@@ -85,12 +83,12 @@ export function updateRectangleSketch(
   y: number,
   tag: string
 ) {
-  ;((pipeExpression.body[2] as CallExpression)
+  ;((pipeExpression.body[1] as CallExpression)
```

Comment:
Indexes changed for the rectangle tool to be one lower because used to be
because there's one less callExpression at the start of the pipeExpression since `startSketchOn` is pulled out.